src/java/main/org/apache/zookeeper/server/NettyServerCnxn.java

```diff
@@ -71,7 +71,7 @@
     NettyServerCnxnFactory factory;
     boolean initialized;
     
-    NettyServerCnxn(Channel channel, ZooKeeperServer zks, NettyServerCnxnFactory factory) {
+    public NettyServerCnxn(Channel channel, ZooKeeperServer zks, NettyServerCnxnFactory factory) {
```

Comment:
Interesting. For me it actually is:

2016-12-05 11:11:02,523 [myid:] - INFO  [Time-limited test:JUnit4ZKTestRunner$LoggedInvokeMethod@98] - TEST METHOD FAILED testNettyRunTimeException
**java.io.IOException: Couldn't instantiate org.apache.zookeeper.server.NettyServerCnxnFactory
        at org.apache.zookeeper.server.ServerCnxnFactory.createFactory(ServerCnxnFactory.java:141)
        at org.apache.zookeeper.server.ServerCnxnFactory.createFactory(ServerCnxnFactory.java:157)
        at org.apache.zookeeper.server.ServerCnxnFactory.createFactory(ServerCnxnFactory.java:151)**